src/Parser/Core/Modules/ManaValues.js

```diff
@@ -26,9 +37,29 @@ class ManaValues extends Analyzer {
             max: max,
             used: manaCost,
           });
+          // The variable 'max' is constant but can differentiate by racial/items.
+          this.baseMana = max;
         });
     }
   }
+
+  suggestions(when) {
+    if(this.combatants.selected.spec.role !== ROLES.HEALER) {
```

Comment:
I think this should be in the on_initialized method, with this suggestion being its own Analyzer (so adding a new analyzer with just this suggestion and this.active set with this conditional). More SRP.